test/odbc/test_pg_datatype_bigint.cpp

```diff
@@ -0,0 +1,361 @@
+#include <gtest/gtest.h>
+#include <sqlext.h>
+#include "odbc_handler.h"
+#include "query_generator.h"
+
+using std::pair;
+
+const string TABLE_NAME = "master_dbo.bigint_table";
```

Comment:
In the pipeline, the odbc tests are run with a fresh instance isolated from other tests so I the only naming conflict we would have would be from our own tests. But we can add the _odbc_test suffix as well just in case.

I added the master_dbo schema as that's connected to the Babelfish endpoint so we would closely imitate a table inside the master database in the dbo schema. Probably doesn't make a different tbh though.